SmartDeviceLink/SDLListFilesOperation.m

```diff
@@ -59,7 +59,15 @@ - (void)sdl_listFiles {
         NSUInteger bytesAvailable = listFilesResponse.spaceAvailable != nil ? listFilesResponse.spaceAvailable.unsignedIntegerValue : 2000000000;
 
         if (weakSelf.completionHandler != nil) {
-            weakSelf.completionHandler(success, bytesAvailable, fileNames, error);
+            if([response.resultCode isEqualToEnum:SDLResultDisallowed]) {
+                NSMutableDictionary *results = [error.userInfo mutableCopy];
+                results[@"resultCode"] = SDLResultDisallowed;
+                NSError *resultError = [NSError errorWithDomain:error.domain code:error.code userInfo:results];
+                weakSelf.completionHandler(success, bytesAvailable, fileNames, resultError);
+            }
+            else {
```

Comment:
Follow the formatting of the project with if / else

Reply:
fixed